src/UIComponents/SaveModel.jsx

```diff
@@ -229,7 +228,6 @@ export default connect(
     trainedModelDetails: state.trainedModelDetails,
     labelColumn: state.labelColumn,
     columnDescriptions: getSelectedColumnDescriptions(state),
-    saveStatus: state.saveStatus,
```

Comment:
cleaning up... this prop wasn't being used in this component, so I deleted it.

Reply:
Thank you for catching this!